pkg/eventing/v1/client.go

```diff
@@ -309,11 +315,17 @@ type BrokerBuilder struct {
 
 // NewBrokerBuilder for building broker object
 func NewBrokerBuilder(name string) *BrokerBuilder {
-	return &BrokerBuilder{broker: &eventingv1.Broker{
+	broker := &eventingv1.Broker{
```

Comment:
`NewBrokerBuilder` is only called for `kn broker create` which does not offer a yaml/json output. So even when the GVK is missing there, this is not an issue as by K8s API conventions this is not harmful (and even expected) that GVK is missing on objects that are sent to the API server.

Same for `NewTriggerBuilder` above.

Reply:
`NewBrokerBuilder` is also used in the tests to create fake response data for the mock client recorder. This is why the tests fail as soon as I remove the GVK.